py/bitbox02/bitbox02/bitbox02/bootloader.py

```diff
@@ -94,6 +107,17 @@ def versions(self) -> typing.Tuple[int, int]:
         firmware_v, signing_pubkeys_v = struct.unpack("<II", response[:8])
         return firmware_v, signing_pubkeys_v
 
+    def hardware(self) -> Hardware:
```

Comment:
When adding the same API to the Go package I realized that this endpoint could return more HW info in the future, so having Hardware==Securechip like here will be a breaking change when that happens. It's not that important as I think there is no production use of the bootloader Python API, but technically Hardware should be an object (TypedDict?) with a SecureChipModel enum inside.

https://github.com/BitBoxSwiss/bitbox02-api-go/pull/116/

Reply:
Done, I also updated the check of the response to print unexpected values, so it behaves the same as your PR